README.md

```diff
@@ -158,3 +157,17 @@ To develop the component, you first need to link it to a Gatsby project. This is
 - [x] Text Confirmation
 - [ ] Page Change
 - [ ] Redirect
+
+## Ignitro Notes Below
+### To Do - Ignitro
+# Field Validation
```

Comment:
Please combine with the above Field Components. In case someone else wants to pick these features up.